admin_tests/pre_reg/test_views.py

```diff
@@ -25,7 +25,8 @@
 
 
 class TestDraftListView(AdminTestCase):
-    def setUp(self):
+    @mock.patch('admin.pre_reg.views.DraftRegistration.get_metadata_files')
```

Comment:
I still think it would be good to not mock it. You're preventing the testing of possible live behavior by mocking it. 

If you don't need to have file data in the draft (as many users might), not having file data in the draft shouldn't break the submit process. This may mean better error handling in `get_metadata_files` is required (it looks like `KeyErrors` might be raised without files).